src/sql/parts/notebook/notebook.component.ts

```diff
@@ -526,6 +424,10 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		provider.bindTo(this.contextKeyService);
 	}
 
+	public async save(): Promise<boolean> {
```

Comment:
Seems like you don't need this as you implemented it in MainThreadNotebookDocumentsAndEditors with a direct call through to the input. I'd suggest removing.